ocaml/xenopsd/lib/xenops_migrate.ml

```diff
@@ -42,7 +42,7 @@ module Handshake = struct
     if n < len then really_read fd buf (ofs + n) (len - n)
 
   (** Receive a 'result' from the remote *)
-  let recv ?(verbose = false) (s : Unix.file_descr) : result =
```

Comment:
Did you mean to leave verbose-mode on? You may have done this for testing, and may get overly spammy.

Reply:
I'll dial it back. It is more important during development.